aw_qt/manager.py

```diff
@@ -138,8 +138,10 @@ def __init__(self, testing: bool=False) -> None:
             # "aw-watcher-network"
         ]
 
-        # TODO: Filter away all modules not available on system
-        self.modules = {name: Module(name, testing=testing) for name in _possible_modules}
+        self.modules = []
+        for name in _possible_modules:
+            if not _locate_executable(name):
```

Comment:
Maybe log an error if a module couldn't be located?

Reply:
Travis fails on type checking, it seems that the code expects `self.modules` to be a dict.